src/fal/cli/args.py

```diff
@@ -136,6 +136,17 @@ def _add_experimental_python_models_option(parser: argparse.ArgumentParser):
     )
 
 
+def _add_experimental_threading(parser: argparse.ArgumentParser):
+    parser.add_argument(
+        "--experimental-threads",
```

Comment:
Note: We might call this `--threads=...` in the future (or `--workers` etc.), this is just the name for us (to test).

Reply:
We already have a `--threads` flag

```
  --threads THREADS     Specify number of threads to use while executing Python scripts and dbt models. Overrides settings in profiles.yml.
```

Reply:
Ah, good to know! We might use this and maybe change the description. It depends whether we want to enable the new parallelization code (and the complete refactor of the stuff) by default on the initial release though.